airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1789,6 +1799,16 @@ components:
           $ref: "#/components/schemas/DestinationDefinitionId"
         dockerImageTag:
           type: string
+    DestinationDefinitionReadWithJobInfo:
+      type: object
+      required:
+        - destinationDefinitionRead
```

Comment:
discussed this offline and the definition read here and in source definition read will no longer be required. they will not be present in failure cases.

Reply:
Made this change.